change/@azure-msal-common-76a4fd19-0905-46ee-b183-fae2ad84478f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
```suggestion
  "type": "minor",
```

Reply:
@jasonnutter I made the change.

Reply:
@jasonnutter when is the release? I would have liked to include this PR